help/en/html/doc/Technical/TechRoadMap.shtml

```diff
@@ -307,16 +258,16 @@
       jlink from Java 9 or later.
 
       <p>
-      While we don't know yet when JMRI will have to move past Java 8, 
-      we do know that the earliest Java 9 or later will be required for users is Fall 2020.
+      While we don't know yet when JMRI development will move past Java 8, 
+      we do know that the earliest that will happen  is after Fall 2020.
       Allowing for migration and
-      development time, and depending on development plans, JMRI's December 2020 release
-      might require Java 11 (in which case it would be called JMRI 5.1)
-      or stay with requiring Java 8 or later (in which case it would be called JMRI 4.20).
-      Releases before Fall 202 are intended to stay fully compatible Java 8.
+      development time, and depending on development plans, 
+      some future JMRI major release (with a number of 5.0.0 or later)
+      will require Java 11 or later.  JMRI updates and minor releases would stay with 
+      Java 8 for a little while after that, but eventually would have go shift over.
```

Comment:
As I wrote on the JMRI mailing list, I think it would be good to say that we expect to move to Java 11 summer 2021.

Reply:
How's this look now?  I'm trying to capture the uncertainty of when in Spring things would really change.

Reply:
I approve this.

Reply:
"but eventually would have go shift over."
I don't believe everyone will be familiar with this form of colloquial English.
The gist of it is fine.

Reply:
@Pugwash1 
The text has been updated. The new text is:
```
While we don't know with certainty when JMRI development will move past Java 8, 
we expect that it will be during the Spring 2021 development releases, 
leading to the June 2021 production releases requiring Java 11 and 
therefore being called JMRI 5.0.
There might be one or two JMRI updates and minor releases early
in Spring that would still work with Java 8.
```